packages/expo/bin/cli

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env bash
+#!/usr/bin/env node
 
-# Forward all arguments to the local CLI (@expo/cli).
-npm exec --no-install -- expo-internal "$@"
```

Comment:
Are there any other remaining references to expo-internal?

Reply:
No, this was the only one within `expo`. `@expo/cli` still exports the binary as `expo-internal` which might still be useful to keep around for dev.